app/design/adminhtml/default/default/layout/main.xml

```diff
@@ -77,7 +77,6 @@ Default layout, loads most of the pages
             <block type="core/text_list" name="notifications" as="notifications">
                 <block type="adminhtml/notification_baseurl" name="notification_baseurl" as="notification_baseurl" template="notification/baseurl.phtml"></block>
                 <block type="adminhtml/cache_notifications" name="cache_notifications" template="system/cache/notifications.phtml"></block>
-                <block type="adminhtml/checkout_formkey" name="checkout_formkey" as="checkout_formkey" template="notification/formkey.phtml"/>
```

Comment:
I kept it because somebody could still disable it via config, should we remove that?